themes/rad/layouts/index.html

```diff
@@ -4,25 +4,24 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Radicle a decentralized, collaborative revision control system built on IPFS.
+        Write open source software without giving your data to third parties or setting up a server.
       </p>
     </div>
     <div id="term">
       <h2>Terminal-first</h2>
       <p>
-        A Radicle project contains a git repository as well as the
-        issues and proposals associated with that repo. All accessible from
-        your terminal.
+        A Radicle project contains a git repository, plus the associated
+        issues and proposals. Access all the conversations
+        that happen around your codebase from your command line.
       </p>
     </div>
     <div id="prog">
       <h2>Programmable</h2>
       <p>
-        Ever wanted to tweak your code collaboration service? With Radicle each 
-        unit of functionality &ndash;<em>a machine</em>&ndash; is its
-        own litte P2P program, written in the Radicle language. You can change
-        them to suit your needs, or create entirely new ones. More on this soon.
+        Each Radicle utility is a P2P program we call a Radicle <em>machine</em>.
+        Need a custom issues system? Modify our version to suit your needs, or create an entirely new utility.
```

Comment:
The connection between the first sentence and the second is a little unclear.

Reply:
Also, "custom issue system" makes it feel like that's the only thing you can tweak.